Upload the visualization part

hi-T0day opened the pull request:
This is all what I have done so far for the visualization part.

hi-T0day committed:
Upload the visualization part

This is all what I have done so far for the visualization part.

hi-T0day committed:
The script that generates the json file is taken out separately

webstergd reviewed (changes requested):
violation of copyright. cannot accept pull

To fix this, include the original authors comments and copyright. Add a section in the copyright that states your code modifies his original work and list out what you changed.

/*Copyright (c) 2013-2016, Rob Schmuecker
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:

* Redistributions of source code must retain the above copyright notice, this
  list of conditions and the following disclaimer.

* Redistributions in binary form must reproduce the above copyright notice,
  this list of conditions and the following disclaimer in the documentation
  and/or other materials provided with the distribution.

* The name Rob Schmuecker may not be used to endorse or promote products
  derived from this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED. IN NO EVENT SHALL MICHAEL BOSTOCK BE LIABLE FOR ANY DIRECT,
INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.*/

hi-T0day committed:
Add the copyright information

hi-T0day committed:
Detail about my work and his work

hi-T0day committed:
Add credits on the web page

“This webpage is reference Rob Schmuecker's work:
http://bl.ocks.org/robschmuecker/7880033”
Show this sentence in the webpage.

webstergd commented:
How do I use the generate.js script?

hi-T0day committed:
Merge branch 'HolmesProcessing/master' into visualization

hi-T0day committed:
Update the description of generating final score script

hi-T0day committed:
Delete the readme file in the subdirectory
